src/KafkaFlow/Administration/TopicManager.cs

```diff
@@ -0,0 +1,69 @@
+namespace KafkaFlow.Administration
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Confluent.Kafka;
+    using Confluent.Kafka.Admin;
+    using KafkaFlow.Configuration;
+
+    internal class TopicManager : ITopicManager
```

Comment:
maybe we can change the name to ClusterManager and create the AdminClient with a Lazy<> here at the ctor

Reply:
Good idea. 
Since it will be a ClusterManager, I believe we need a Factory like ConsumerManagerFactory.
Ok with it?

Reply:
As we discussed offline, I implemented a ClusterManagerAccessor.